jenkins-design-language/src/js/components/pipeline/PipelineGraph.jsx

```diff
@@ -622,12 +623,12 @@ export class PipelineGraph extends Component {
 
         const visualElements = []; // Buffer for children of the SVG
 
-        this.renderSelectionHighlight(visualElements);
-
         connections.forEach(connection => {
             this.renderCompositeConnection(connection, visualElements);
         });
 
+        this.renderSelectionHighlight(visualElements);
```

Comment:
Why did this need to move?

Reply:
Sorry, I haven't uploaded a screenshot to show the changes, I moved this so the selection highlight is over the white edges and white highlight:
![image](https://user-images.githubusercontent.com/2933351/42426675-1adb34b4-836d-11e8-8114-42b92633cfea.png)

Reply:
Ah, nice.